### DISCUSSION TITLE
Not support Figures.vue from vuepress?

### DISCUSSION THREAD

**Discussion by Zhao-Michael** [vuepress-theme-hope/vuepress-theme-hope#3786] [Q&A] [answered]
我看很多 vuepres 项目中都有 Figure.vue 这个组件

例如 https://github.com/zhengda/The-Mythical-Man-Month-zh/blob/main/docs/.vuepress/components/Figures.vue

vuepress-theme-hope 好像没有默认支持这个组件？或者是我不知道怎么在 vuepress-theme-hope 里使用它😓

我现在导入这种项目的时候都是手动更新 (我感觉我肯定是哪里没用对😅) ：
`<Figures figure="2-1">Eisenhower matrix</Figures>           =>          ![Eisenhower matrix](/figures/ch2/fg2-1.jpg)`

Thx in advance~

**Comment by Mister-Hope** [comment]
并不是很清楚你在说什么

**Reply by Zhao-Michael** [reply]
有 markdown 代码，只能用图片回复了

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/15730936/66392cdb-495c-4fc6-8855-a44a62428aaf)

**Reply by Mister-Hope** [reply]
Why should my theme support a custom component like this? It's your own responsibility to write a component to achieve similar things.

**Reply by Zhao-Michael** [reply]
显然，我的原意并没希望在vuepress-theme-hope新增支持这个 Figures 组件。。。。

**Comment by Mister-Hope** [comment, marked as answer]
- https://theme-hope.vuejs.press/zh/guide/component/sfc.html
- https://theme-hope.vuejs.press/zh/guide/component/global.html
